Automattic/wp-calypso
# SSO Invite: Move invite status under invite details

Related to #35278

## Proposed Changes

* These changes move the invite status underneath the invite details, allows the email/username to overflow and fixes the button overlapping issues on mobile viewport.

### Before:

Desktop:
<img width="764" alt="CleanShot 2024-01-27 at 11 52 16@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/d3e1d11e-f25e-470c-8107-9133ed96870f">

Mobile:
<img width="390" alt="CleanShot 2024-01-29 at 09 47 30@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/f6f3a315-08ce-47df-a2d5-d78951e7ccf1">


### After: 

Desktop:
<img width="753" alt="CleanShot 2024-01-27 at 11 57 13@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/ac36289c-f763-480c-a904-a2c7af6d878d">

Mobile:

<img width="403" alt="CleanShot 2024-01-29 at 09 49 42@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/b2505b2f-1211-4fa4-a00c-9fb2e40cadb1">


## Testing Instructions



* Checkout branch or use calypso live link
* Navigate to `http://calypso.localhost:3000/people/team/{SITE_SLUG}`
* Add/Invite a new user to your site, if you haven't already.
* Click on the new pending invite to see more details about the user and the invitation.
* Test the "Resend Invite" and "Revoke" buttons.
* The "Resend Invite" button text should change to "Invite Sent!" after a successful invite request has been made.
* The "Revoke" button should display an "Invite deleted" notice after the invite has been revoked and you should be navigated to the previous page.

## Pre-merge Checklist
